burnman/solutionmodel.py

```diff
@@ -13,32 +13,34 @@
 # python interpreter
 try:
     import os
-    if 'NUMBA_DISABLE_JIT' in os.environ and int(os.environ['NUMBA_DISABLE_JIT']) == 1:
+    if (('NUMBA_DISABLE_JIT' in os.environ
+         and int(os.environ['NUMBA_DISABLE_JIT']) == 1)):
         raise ImportError("NOOOO!")
-    from numba import jit
+    from numba import njit
 except ImportError:
-    def jit(fn):
+    def njit(fn):
         return fn
 
 
-@jit
-def _ideal_activities_fct(molar_fractions, endmember_occupancies, n_endmembers, n_occupancies, site_multiplicities, endmember_configurational_entropies):
+def _ideal_activities_fct(molar_fractions, endmember_occupancies, n_endmembers,
+                          n_occupancies, site_multiplicities,
+                          endmember_configurational_entropies):
     site_occupancies = np.dot(molar_fractions, endmember_occupancies)
     a = np.power(site_occupancies,
                  endmember_occupancies * site_multiplicities).prod(-1)
-    normalisation_constants = np.exp(endmember_configurational_entropies /
-                                     constants.gas_constant)
+    normalisation_constants = np.exp(endmember_configurational_entropies
+                                     / constants.gas_constant)
     return normalisation_constants * a
 
-@jit
```

Comment:
why did you remove this?

Reply:
Because a) jit is deprecated (http://numba.pydata.org/numba-doc/latest/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit), b) these functions are already vectorised, so njit doesn't help much, and c) the rewrite is really fiddly, because numba doesn't yet allow operations over specific axes.